arrows/kpf/yaml/CMakeLists.txt

```diff
@@ -42,19 +38,19 @@ set( private_sources
   kpf_bounce_buffer.cxx
 )
 
-kwiver_add_library( vital_kpf
+kwiver_add_library( kpf_yaml
   ${sources}
   ${public_headers}
   ${private_sources}
 )
 
-target_include_directories(vital_kpf PRIVATE ${YAML_CPP_INCLUDE_DIR})
+target_include_directories(kpf_yaml PRIVATE ${YAML_CPP_INCLUDE_DIR})
```

Comment:
If kfp_yaml is including <yaml-cpp/yaml.h> in any of its headers, this include_directories needs to be PUBLIC.

Reply:
Done!